src/venue.md

```diff
@@ -30,5 +30,13 @@ title: "Venue"
 
   <p>The venue is accessible both by BTS (Victory Monument) and the Airport Link (Phaya Thai).</p>
 
-  <a href="https://goo.gl/maps/vnLpSxt1zzw" target="_blank" class="card-venue__btn btn btn--primary">View in Google Maps</a>
+  <a href="https://goo.gl/maps/vnLpSxt1zzw" target="_blank" class="btn btn--primary">View in Google Maps</a>
+</section>
+
+<section class="venue-booking">
+  <h2>Staying There</h2>
+
+  <p>If you are interested in staying in the hotel during the event, you can use a dedicated booking portal provided directly by the hotel. The service is available to all attendees.</p>
+
+  <a href="https://www.idem.events/r/rubyconfth-2022" target="_blank" class="btn btn--primary">Book a Room</a>
```

Comment:
I got into the habit of always adding `rel="noreferrer noopener"` to `target="_blank"`. While `noreferrer` is probably unnecessary for this, I think `noopener` could still make sense.

Reply:
Good point. I checked the status on how browsers handle this nowadays, and it seems that with `target="_blank"`, `noreferrer` is implied indirectly. 

![image](https://user-images.githubusercontent.com/696529/193743096-eb281846-f8ed-46c5-8128-b61b83137321.png)

Apart from Opera and Android Webviews, the browser support is good in this implicit behavior: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#browser_compatibility. So I think we can leave it as-is in this case :-)